interceptor/pkg/interceptor/pdinterceptor.go

```diff
@@ -90,15 +90,36 @@ func (pdi *PagerDutyInterceptor) executeInterceptor(r *http.Request) ([]byte, er
 	}
 	r.Body = io.NopCloser(bytes.NewReader(body.Bytes()))
 
+	var wrapper struct {
```

Comment:
naming things is hard :D I think this is okay, but can we highlight that this is the "original" request reaching the evenlistener from pagerduty we are unwrapping

or a comment on what its for, would achieve the same :)